resources/assets/js/home.js

```diff
@@ -307,31 +307,9 @@ window.PoliceScorecardHome = window.PoliceScorecardHome || (function () {
 
         fetch(url).then(res => res.json()).then((out) => { sheriffData = out; createViz(lookupTable); }).catch(err => { throw err });
 
-        function createViz(lookupTable) {
-
-            const lookupData = filterLookupTable(lookupTable);
-
-            // Filters the lookup table to features with the 'US' country code
-            // and keys the table using the `unit_code` property that will be used for the join
-            function filterLookupTable(lookupTable) {
-
-                let lookupData = {};
-
-                for (layer in lookupTable) {
-                    for (worldview in lookupTable[layer].data) {
-                        for (feature in lookupTable[layer].data[worldview]) {
-                            let featureData = lookupTable[layer].data[worldview][feature];
-
-                            // Filter the lookup data for the US
-                            if (featureData.iso_3166_1 == 'US') {
-                                // Use `unit_code` property that has the FIPS code as the lookup key
-                                lookupData[featureData['unit_code']] = featureData;
-                            }
-                        }
-                    }
-                }
-                return lookupData;
-            }
+        const createViz = (table) => {
+            // Create Lookup Table for MapBox Boundaries
+            const boundaries = filterLookupTable(table);
```

Comment:
nit pick suggestion hehe

```suggestion
            const fipsBoundaries = filterLookupTable(table);
```